### DISCUSSION TITLE
Download Twitter Media by Hashtags/Username Mentions

### DISCUSSION THREAD

**Discussion by trumbladome** [AAndyProgram/SCrawler#68] [Ideas]
I have got the profile tool downloading but what i what like to do is download Videos (could include pictures) of media that has a specific hashtag or username mentioned this could be defined to a last 24 hours/48 hours/week/month etc

For example Avril Lavigne videos that mentions her twitter account
https://twitter.com/search?q=%40avrillavigne&src=typed_query&f=video

**Comment by AAndyProgram** [comment]
Maybe in the future. SCrawler is developed as a user downloader with advanced user management. So there's nothing like that here. It requires time to figure out how to do it and have the desire. Currently, I don't have either.

**Reply by AAndyProgram** [reply]
This will not be implemented, sorry.
